src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -73,6 +77,12 @@
 
 	private long lastInventoryUpdate;
 
+	/**
+	 * Instantiates a new Pokemon go.
+	 *
+	 * @param auth   the auth
+	 * @param client the client
+	 */
```

Comment:
Altough for public API it is good to have documentation, but do you think this gives anything more than the actual code?

looking at the constructor(given that everyone uses IDE nowadays), you can see it's a pokemongo class, takes authinfo and okhttpclient